.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+---
+name: CI
+on: [pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.5', '3.6', '3.7', '3.8', '3.9', '3.10']
```

Comment:
Everything below Python 3.7 is EOL. I suggest to stick to supported Python releases.

Reply:
Yeah, that'd be sensible. I don't know what the maintainers want to be supporting, though?